### PR TITLE
Integrations landing page

### PR DESCRIPTION
**Pull request by radekkaluzik**
### Description
Added new Overview tab based on mocks, integrated already existing IntegrationsDropdown component

[RHCLOUD-33731](https://issues.redhat.com/browse/RHCLOUD-33731)

---

#### Before:

image

#### After:

image

---

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [x] Change reviewed for extraneous code 
- [x] UI best practices adhered to 
- [x] Commits squashed and meaningfully named 
- [x] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

File: src/components/Icons/RedhatIcon.js
Comment (aferd) [inline src/components/Icons/RedhatIcon.js:1]: empty file? we should probably delete if not using.

File: src/components/Icons/ReportingIcon.js
Comment (aferd) [inline src/components/Icons/ReportingIcon.js:1]: empty file? we should probably delete if not using.

File: src/components/Overview/index.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+                  defaultMessage: 'Get started with Integrations',
+                })}
+              </Title>
+            </CardTitle>
+            <CardBody>
+              <p className="pf-u-mb-md">
+                {intl.formatMessage({
+                  id: 'integrations.overview.heroParagraph',
+                  defaultMessage:
+                    'Notifications and integrations services work together to transmit messages to third-party applicationendpoints, such as instant messaging platforms and external ticketing systems, when triggering events occur.',
```
[the remark this answers is not in the document]
Comment (radekkaluzik) [reply src/components/Overview/index.js]: fixed (copy pasting from the mocks)

File: src/components/Icons/RedhatIcon.js
Comment (radekkaluzik) [reply src/components/Icons/RedhatIcon.js:1]: Deleted the entire Icons folder. It is not needed anymore.

File: src/components/Overview/CustomDataListItem.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  const toggleExpand = () => setIsExpanded((prev) => !prev);
+
+  return (
+    <React.Fragment>
+      <DataListItem aria-labelledby="item1" isExpanded={isExpanded} className={isExpanded && 'active-item'}>
+        <DataListItemRow className="pf-u-align-items-center">
+          <DataListToggle isExpanded={isExpanded} id="toggle1" aria-controls="expand1" onClick={toggleExpand} />
+          <DataListItemCells
+            dataListCells={[
+              <DataListCell key={'cell-' + icon.toString().toLowerCase()}>
```
Comment (fhlavac) [inline src/components/Overview/CustomDataListItem.js:41]:
```suggestion
              <DataListCell key={`cell-${icon.toString().toLowerCase()}`}>
```

File: src/components/Overview/index.js [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+          </Tr>
+        </Tbody>
+      </Table>
+
+      <a href={`${isBeta() ? '/preview' : ''}/settings/learning-resources`} className="pf-u-mb-lg">
+        {intl.formatMessage({
+          id: 'integrations.overview.viewAllLearningResources',
+          defaultMessage: 'View all Settings learning resources',
+        })}
+      </a>
```
Comment (fhlavac) [inline src/components/Overview/index.js:333]: Can we use PF link button instead?
Also with the killed `/preview`, we may not need the changes in the URL

File: src/components/Overview/index.js [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+                  e.preventDefault();
+                  window.open(VIEW_DOCUMENTATION, '_blank');
+                }}
+              >
+                {intl.formatMessage({
+                  id: 'integrations.overview.recommendedContentTableLink1',
+                  defaultMessage: 'View documentation',
+                })}{' '}
+                <ExternalLinkAltIcon />
+              </a>
```
Comment (fhlavac) [inline src/components/Overview/index.js:297]: Can we use PF link button instead?

File: src/components/Overview/index.js [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+              </Title>
+            </CardTitle>
+            <CardBody>
+              <p className="pf-u-mb-md">
+                {intl.formatMessage({
+                  id: 'integrations.overview.heroParagraph',
+                  defaultMessage:
+                    'Notifications and integrations services work together to transmit messages to third-party application endpoints, such as instant messaging platforms and external ticketing systems, when triggering events occur.',
+                })}
+              </p>
```
Comment (fhlavac) [inline src/components/Overview/index.js:173]: Could be replaced with Text

File: src/components/Overview/CustomDataListItem.js [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+              e.preventDefault();
+              window.open(learnMoreLink, '_blank');
+            }}
+          >
+            {intl.formatMessage({
+              id: 'integrations.overview.dataListItemLearnMore',
+              defaultMessage: 'Learn more',
+            })}{' '}
+            <ExternalLinkAltIcon />
+          </a>
```
Comment (fhlavac) [inline src/components/Overview/CustomDataListItem.js:95]: Could be replaced with PF components

**Comment by fhlavac** [comment]
@radekkaluzik just some nitpicky comments. It looks good! 🎉

File: src/components/Overview/CustomDataListItem.js (hunk 2 above)
Comment (radekkaluzik) [reply src/components/Overview/CustomDataListItem.js:41]: fixed

File: src/components/Overview/index.js (hunk 5 above)
Comment (radekkaluzik) [reply src/components/Overview/index.js:173]: Fixed

File: src/components/Overview/index.js (hunk 4 above)
Comment (radekkaluzik) [reply src/components/Overview/index.js:297]: Fixed

File: src/components/Overview/CustomDataListItem.js (hunk 6 above)
Comment (radekkaluzik) [reply src/components/Overview/CustomDataListItem.js:95]: Fixed

File: src/components/Overview/index.js (hunk 3 above)
Comment (radekkaluzik) [reply src/components/Overview/index.js]: Fixed

**Review by fhlavac — APPROVED** [review]

**Comment by fhlavac** [comment]
🥳

**Pull request merged**